CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -161,10 +161,10 @@ public function buildQuickForm() {
     //get the frequency units.
     $this->_freqUnits = CRM_Core_SelectValues::getRecurringFrequencyUnits();
 
-    $numericOptions = CRM_Core_SelectValues::getNumericOptions(0, 30);
-
     //reminder_interval
-    $this->add('select', 'start_action_offset', ts('When'), $numericOptions);
+    $this->add('number', 'start_action_offset', ts('When'), array('style' => 'width:5em'));
```

Comment:
I did add 'min' => 1, but later noticed the select controls actually have 0-30, so changed min to zero.